.travis.yml

```diff
@@ -13,7 +13,7 @@ matrix:
     - php: 7.0
       env: PHPUNIT_TEST=admin,cwpcms,Default
     - php: 7.1
-      env: PHPCS_TEST=1
+      env: PHPUNIT_TEST=core,cwpcore,admin,cwpcms,Default,cwpsearch PHPCS_TEST=1
```

Comment:
It's fairly likely that running all of these together will time out in Travis, but let's see :D

Reply:
I'd originally left it out because all the tests are fully covered in segments by the other environments. I figured why not run a code sniff since it's otherwise only testing installation :)

Reply:
Yep, if it's all green then that's OK but I suspect there's be some test failures in this, for example https://www.cwp.govt.nz/developer-docs/en/1.8/releases_and_changelogs/cwp_recipe_basic_1.8.0/#accepted-failing-tests-2 which are mostly test failures affected by the global state

I guess we'll deal with this happening as we go, but ideally we'd have green CI builds. We could probably try and isolate those tests where possible.